src/framework/mpas_io.F

```diff
@@ -5033,6 +5037,149 @@ subroutine MPAS_io_get_att_real1d(handle, attName, attValue, fieldname, precisio
 
    end subroutine MPAS_io_get_att_real1d
 
+   function handle_put_att_pio_redef(handle) result (pio_ierr)
```

Comment:
If these new routines are only used by PIO, can we enclose them in an `MPAS_PIO_SUPPORT` pre-processing directive?